src/Tab.cpp

```diff
@@ -31,6 +31,9 @@ void Tab::startup() {
     header.setOnClickCallback([this](int id) {
         this->onClickCallback(id);
     });
+    header.setOnRemoveCallback([this](int id) {
+        this->onRemoveTabCallback(id);
```

Comment:
Esse classe deveria ter um metodo onRemoveTabCallback, igual o do onClickCallback(linha 11 desse arquivo).
Se tu for ver, no onClickCallback, eu uso o id da Tab pra passar pro callback que mainWindow setou nessa Tab